packages/fx-core/src/component/configManager/lifecycle.ts

```diff
@@ -241,21 +242,47 @@ export class Lifecycle implements ILifecycle {
             ","
           )}) found for Action ${this.stringifyDriverDef(driver)} in lifecycle ${this.name}`
         );
-        summaries.push([
-          `${SummaryConstant.Failed} Unresolved placeholders: ${unresolved.join(",")}`,
-        ]);
-        return {
-          result: err({
-            kind: "PartialSuccess",
-            env: envOutput,
-            reason: {
-              kind: "UnresolvedPlaceholders",
-              failedDriver: driver,
-              unresolvedPlaceHolders: unresolved,
-            },
-          }),
-          summaries,
-        };
+
+        // Special handling for file/createOrUpdateEnvironmentFile driver
+        // Let user input OpenAI environment variables if unresolved placeholders are found
+        if (driver.uses == "file/createOrUpdateEnvironmentFile") {
```

Comment:
Suggest implement this logic in prerequisite checker. ConfigManager should not be aware of the content of the action.

Reply:
As env variables need to be analyzed from yaml + env file, it's not that isolated. So cannot move to prerequisite checker.